generated/api/v2/common.swagger.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Any idea why we now have a `common.swagger.json` and a `common_compliance.swagger.json`?  They don't seem to provide much and weren't generated before.  Curious if these are required or if there is a flag to not build them.

Reply:
Nice find! 

I looked at this. I think this comes from the `make` target and how files are provided to the generator. For `v1` - we are using `$(dir $<)/*.proto`, but for `v2` we use `$<`. Link here: https://github.com/stackrox/stackrox/blob/7d3715ec0f7b62684366499b25ee34b861440df3/make/protogen.mk#L271

This is now consolidated, and all generator commands are using `$(dir $<)/*.proto`.